nest/src/main.ts

```diff
@@ -0,0 +1,18 @@
+import {NestFactory} from '@nestjs/core';
+import {AppModule} from './app.module';
+import {RmqOptions, Transport} from '@nestjs/microservices';
+import {ConfigService} from '@nestjs/config';
+
+async function bootstrap() {
+    const app = await NestFactory.create(AppModule);
+    const config = app.get(ConfigService);
+
+    app.connectMicroservice<RmqOptions>({
+        transport: Transport.RMQ,
+        options: config.get('rabbitmq'),
+    });
```

Comment:
Only one? Thought we defined two?

Reply:
This is the server, the client is registered somewhere else.